graph_altimeter/scan/iam/README.md

```diff
@@ -21,10 +20,29 @@ the Neptune client.
 ## Determining the name of the AWS resources affected by an action
 
 The file: `iam_definition.json` contains the information about all the possible
-actions in a policy document and the names of the resources they affect to
-for a given AWS service.
+actions in a policy document and the names of the affected resources for a
+given AWS service.
 For instance, the action `DeleteTable` for the AWS Service: `dynamodb`
 affect to the resource `table`
 
-The file is copied from the utility: [parliament](https://github.com/duo-labs/parliament) and must be updated using
-the the tool defined in that repository.
+The file is genrerated by a [tool] from the awesome [Parliament] utility.
+
+### Updating the iam_definitions.json file
+
+Execute the following commands (notice yo will need Python3 and virtualenv).
```

Comment:
If we apply the recommended suggestions we can skip the clarification about python3 and virtualenv).

Reply:
👍